Special abilities PR 5, Split Speed values (#8)
GameBrains/er-core: modules/special-rules/pages/brood-telepathy.adoc

---

> Apply these rules to detachments with the {brood-telepathy} ability:

Reviewer: ```suggestion
Where your detachment has the {brood-telepathy} ability:
```
